.github/workflows/release.yaml

```diff
@@ -90,25 +90,35 @@ jobs:
 
       - uses: ko-build/setup-ko@v0.6
         with:
-          version: v0.14.1
+          version: v0.15.1
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.2.0
+        with:
+          cosign-release: "v2.2.2"
 
       - name: Prepare
         run: |
           echo "BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
+          echo "REPOS"="quay.io/grafana-operator/grafana-operator" "ghcr.io/${{ github.repository }}" >> $GITHUB_ENV
```

Comment:
nit-pick: we should also .lower() the `github.repository`, in my testing I started getting failures due to my repo being `HubertStefanski` rather than `hubertstefanski`.

However, this really isn't an issue since I doubt we'll be moving the operator anywhere else at this point, and releasing from a personal repo just isn't a use case we need to support, but thought I'd share just in case anyone comes up on this in the future.